presto-product-tests/README.md

```diff
@@ -0,0 +1,37 @@
+# Presto product tests
+
+## Test configuration
+
+Test assume that you have hadoop and presto cluster running. For setting up development environment we
```

Comment:
Since this test suite requires that a presto cluster already exist, running it as part of the maven build seems like the wrong approach. I would expect this module to produce a binary artifact that bundles all the tests and that can be used to validate a presto instance.

I.e., the process would be:
- run the build, this produces the presto binary and test suite binary
- deploy the presto binary to a cluster
- run test suite binary against the cluster